src/main/java/org/swisspush/redisques/util/QueueStatisticsCollector.java

```diff
@@ -0,0 +1,361 @@
+package org.swisspush.redisques.util;
+
+import static org.swisspush.redisques.util.RedisquesAPI.*;
+
+import io.vertx.core.eventbus.Message;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.redis.client.RedisAPI;
+import io.vertx.redis.client.Response;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.stream.Collectors;
+import org.swisspush.redisques.lua.LuaScriptManager;
+
+/**
+ * Class StatisticsCollector helps collecting statistics information about queue handling and
+ * failures.
+ * <p>
+ * Due to the fact that there is a Redisques responsible for one queue instance at the same time
+ * in a cluster, we could assume that the values are good enough when cached locally for queue
+ * processing by itself. If the responsible Redisques Instance changes, it will build up the
+ * statistics straight away by itself anyway.
+ * <p>
+ * But in the case of the statistics read operation, we must get the statistics of all queues
+ * existing, not only the ones which are treated by the redisques instance (there might be multiple
+ * involved). Therefore, we must write the statistics values as well to redis and retrieve them from
+ * there once needed. Note that the statistics is written asynch and only in the case of queue
+ * failures, therefore it shouldn't happen too often.
+ */
+public class QueueStatisticsCollector {
+
+    private static final Logger log = LoggerFactory.getLogger(QueueStatisticsCollector.class);
+
+    private static final String STATSKEY = "redisques:stats";
+    private final static String QUEUE_FAILURES = "failures";
+    private final static String QUEUE_BACKPRESSURE = "backpressureTime";
+    private final static String QUEUE_SLOWDOWNTIME = "slowdownTime";
+
+    private final Map<String, AtomicLong> queueFailureCount = new HashMap<>();
+    private final Map<String, Long> queueBackpressureTime = new HashMap<>();
+    private final Map<String, Long> queueSlowDownTime = new HashMap<>();
+
+    private final RedisAPI redisAPI;
+    private final LuaScriptManager luaScriptManager;
+    private final String queuePrefix;
+
+    public QueueStatisticsCollector(RedisAPI redisAPI, LuaScriptManager luaScriptManager,
+        String queuePrefix) {
+        this.redisAPI = redisAPI;
```

Comment:
Again, this is what default formatter does!

Reply:
My opinion -> auto-formatters are not a holy thing. They are a nice tool that makes our dev lifes easier. But they are not a replacement for thinking ;)

So my suggestion is to not throw away our brain just because we have a fancy auto-formatter.